linux_os/guide/system/software/system-tools/package_pigz_removed/rule.yml

```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+prodtype: rhel6,rhel7,rhel8,fedora,ol7,ol8
+
+title: 'Uninstall pigz Package'
+
+description: |-
+    {{{ describe_package_remove(package="pigz") }}}
+
+rationale: |-
+    pigz, which stands for parallel implementation of gzip,
+    is a fully functional replacement for gzip that exploits
+    multiple processors and multiple cores to the hilt when compressing data.
```

Comment:
@jan-cerny  This rationale does not clearly explain why this package should be removed. Can you clarify?